src/views/ForgetLogin.vue

```diff
@@ -0,0 +1,98 @@
+<template>
```

Comment:
There are two `ForgetLogin` file. Please remove this one.

Reply:
done